orchestrator/orchestrator/src/main_loop.rs

```diff
@@ -304,7 +355,7 @@ pub async fn eth_signer_main_loop(
                 }
 
                 // sign the last unsigned valsets
-                match get_oldest_unsigned_valsets(&mut grpc_client, our_cosmos_address).await {
+                match get_oldest_unsigned_valsets(&mut grpc_client, cosmos_address.clone()).await {
```

Comment:
Since the cosmos address should probably be immutable in most places we're using it, do we need to be cloning this object or can we adjust to using a reference?

Reply:
Heh, turns out `Address` implements copy, so neither `.clone()` or an explicit borrow are needed